haystack/utils.py

```diff
@@ -29,6 +29,39 @@ def launch_es():
         time.sleep(15)
 
 
+def launch_milvus():
+    # Start a Milvus server
+    # You can start Elasticsearch on your local machine instance using Docker. If Docker is not readily available in
+    # your environment (eg., in Colab notebooks)
+    logger.info("Starting Milvus ...")
+    setup_milvus_config()
+    status = subprocess.run(
+        ['sudo docker run -d --name milvus_cpu_1.0.0 \
+          -p 19530:19530 \
+          -p 19121:19121 \
+          -v /home/$USER/milvus/db:/var/lib/milvus/db \
+          -v /home/$USER/milvus/conf:/var/lib/milvus/conf \
+          -v /home/$USER/milvus/logs:/var/lib/milvus/logs \
+          -v /home/$USER/milvus/wal:/var/lib/milvus/wal \
+          milvusdb/milvus:1.0.0-cpu-d030521-1ea92e'
+        ],
+        shell=True
+    )
+    if status.returncode:
+        logger.warning("Tried to start Milvus through Docker but this failed. "
+                       "It is likely that there is already an existing Milvus instance running. ")
+    else:
+        time.sleep(15)
+
+
+def setup_milvus_config():
```

Comment:
Why having a separate method just for this warning? I'd rather move it into `launch_milvus()` and eventually implement it also there. Let's not pollute utils with too many intertwined helper methods ...